tests/asgi/test_asgi_servers.py

```diff
@@ -109,17 +109,20 @@ def _run_server_isolated(process_factory, host, port):
         server.send_signal(signal.CTRL_C_EVENT)
```

Comment:
Apparently this is supported when "processes started with a creationflags parameter which includes CREATE_NEW_PROCESS_GROUP". I added that flag, but it doesn't seem to make any difference on Travis.

https://docs.python.org/3/library/subprocess.html#subprocess.Popen.send_signal

Reply:
I tried it, and if irrc when starting a process in a new process group the signals are ignored because the current process can only send signals to its group. So what eneded up happening is that only the caller recieves the ctrl_c

Reply:
ctrl_c is ignored by default, unless the child process explicitly enables support for it, ctrl_break works, but the exit code is not 0

Reply:
Thanks! That info was what I was missing.